Update "software" method in Entertainment

phallobst opened the pull request:
See #4410

phallobst committed:
Update "software" method in Entertainment

Carlgo11 commented on _data/entertainment.yml:
```diff
@@ -49,7 +49,7 @@ websites:
       - phone
-      - software
+      - totp
       - hardware
```
```suggestion
      - u2f
```

Carlgo11 commented on _data/entertainment.yml:
```diff
@@ -49,7 +49,7 @@ websites:
       - sms
       - phone
-      - software
+      - totp
```
```suggestion
      - totp
      - proprietary
```

Carlgo11 commented on _data/entertainment.yml:
```diff
@@ -145,8 +145,8 @@ websites:
-      - software
-    doc: https://help.smashcast.tv/customer/en/portal/articles/2779887
+      - totp
+    doc: https://support.desk.com/customer/portal/articles/1894034
```
That doc link is for Salesforce Desk. Looks like Smashcast's help site is down at the moment so let's leave it empty for now.
```suggestion
```

Carlgo11 commented on _data/entertainment.yml:
```diff
@@ -201,6 +201,6 @@ websites:
       - sms
       - phone
-      - software
+      - totp
```
```suggestion
      - totp
      - proprietary
```

Carlgo11 commented on _data/entertainment.yml:
```diff
@@ -201,6 +201,6 @@ websites:
       - phone
-      - software
+      - totp
       - hardware
```
```suggestion
      - u2f
```

phallobst committed:
Update _data/entertainment.yml

Co-Authored-By: Carlgo11 <Carlgo11@users.noreply.github.com>

phallobst committed:
Update _data/entertainment.yml

Co-Authored-By: Carlgo11 <Carlgo11@users.noreply.github.com>

phallobst committed:
Update _data/entertainment.yml

I'm an idiot... I thought they may have been bought by another company, but it's actually their hosting provider :man_facepalming:

Co-Authored-By: Carlgo11 <Carlgo11@users.noreply.github.com>

phallobst committed:
Update _data/entertainment.yml

Co-Authored-By: Carlgo11 <Carlgo11@users.noreply.github.com>

phallobst committed:
Update _data/entertainment.yml

Co-Authored-By: Carlgo11 <Carlgo11@users.noreply.github.com>
